SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -48,23 +49,23 @@ void USpatialGDKSettings::PostInitProperties()
 // Add a pop-up to warn users to update their config upon changing the using QBI property.
 void USpatialGDKSettings::PostEditChangeProperty(FPropertyChangedEvent& PropertyChangedEvent)
 {
-	if (PropertyChangedEvent.Property == nullptr)
+	if (PropertyChangedEvent.Property != nullptr)
 	{
-		return;
-	}
-	const FName PropertyName = PropertyChangedEvent.Property->GetFName();
-
-	if (PropertyName == GET_MEMBER_NAME_CHECKED(USpatialGDKSettings, bUsingQBI))
-	{
-		const EAppReturnType::Type Result = FMessageDialog::Open(EAppMsgType::YesNo,
-			FText::FromString(FString::Printf(TEXT("You must set the value of the \"enable_chunk_interest\" Legacy flag to \"%s\" in your launch configuration file for this to work.\n\nIf you are using an auto-generated launch config, you can set this value from within Unreal Editor by going to Edit > Project Settings > SpatialOS GDK for Unreal > Settings.\n\nDo you want to configure your launch config settings now?"),
-				bUsingQBI ? TEXT("false") : TEXT("true"))));
+		const FName PropertyName = PropertyChangedEvent.Property->GetFName();
 
-		if (Result == EAppReturnType::Yes)
+		if (PropertyName == GET_MEMBER_NAME_CHECKED(USpatialGDKSettings, bUsingQBI))
 		{
-			FModuleManager::LoadModuleChecked<ISettingsModule>("Settings").ShowViewer("Project", "SpatialGDKEditor", "Editor Settings");
+			const EAppReturnType::Type Result = FMessageDialog::Open(EAppMsgType::YesNo,
+				FText::FromString(FString::Printf(TEXT("You must set the value of the \"enable_chunk_interest\" Legacy flag to \"%s\" in your launch configuration file for this to work.\n\nIf you are using an auto-generated launch config, you can set this value from within Unreal Editor by going to Edit > Project Settings > SpatialOS GDK for Unreal > Settings.\n\nDo you want to configure your launch config settings now?"),
```

Comment:
We've removed the bUseQBI to be user facing at all. I believe this message isn't needed anymore.